core/services/workflows/cmd/cre/utils/runner.go

```diff
@@ -100,11 +107,12 @@ var defaultCleanup = func(ctx context.Context, cfg RunnerConfig, registry *capab
 
 func DefaultHooks() *RunnerHooks {
 	return &RunnerHooks{
-		Initialize: defaultInitialize,
-		BeforeRun:  emptyHook,
-		AfterRun:   emptyHook,
-		Cleanup:    defaultCleanup,
-		Finally:    emptyHook,
+		Initialize:  defaultInitialize,
+		BeforeStart: emptyBeforeStart,
+		AfterRun:    emptyHook,
```

Comment:
Why are there three hooks after wait? What's the value in splitting them like this? Doc string for purpose of hook

Reply:
Not sure, those 3 were preexisting, I'm also unclear on the use case but didn't want to change the existing setup without asking the original author

Reply:
Left doc strings up above on the struct regarding what I intended each one to be used for. We _can_ clean a few up since they are unused. Depends how far we plan to go with testing.